region/lowernorfair/east.json

```diff
@@ -1556,13 +1790,13 @@
                     "h_canNavigateHeatRooms",
```

Comment:
this should need something like a wall jump or space jump to get high enough to place the pb that breaks the blocks